### PR TITLE
[FIX JENKINS-39339] contextual try Blue Ocean button

### PR DESCRIPTION
**Pull request by tfennelly**
# Description

See [JENKINS-39339](https://issues.jenkins-ci.org/browse/JENKINS-39339).

ATH tests: https://github.com/jenkinsci/blueocean-acceptance-test/pull/65

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [x] Ran Acceptance Test Harness against PR changes. https://github.com/jenkinsci/blueocean-acceptance-test/pull/65

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@jenkinsci/code-reviewers @reviewbybees

### CODE REVIEW

**Comment by michaelneale** [comment]
Nice tom!

I am looking at links from pipeline screen -> classic config, looks like it will be much smaller than this though (as URLs directly mechanically translate)

File: blueocean-events/src/main/resources/index.jelly [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?jelly escape-by-default='true'?>
+<div>
+  Blue Ocean Events
```
Comment (michaelneale) [inline blueocean-events/src/main/resources/index.jelly]: how come this is needed now?

**Comment by michaelneale** [comment]
Nice work @tfennelly. A fair bit more code than I expected but a lot of it is tests so that is great. 

Running ATH here https://ci.blueocean.io/job/ATH-Jenkinsfile/job/master/261/ (as artifact one didn't seem to pass). 

If passes ATH - :bee: from me, very cool. Tried it out and works well when logged in or not.

**Comment by michaelneale** [comment]
:bee: it passes. 🚢 :shipit:

**Comment by tfennelly** [comment]
@michaelneale yes, by far most of the changes in this PR are wrt tests. As for the ATH, as noted in the description here, there's a branch that includes tests specific to this ... https://ci.blueocean.io/job/ATH-Jenkinsfile/job/JENKINS-39339-contextual-try-bo/2/

**Pull request merged**